custom_components/lutron_caseta_pro/fan.py

```diff
@@ -18,11 +18,11 @@
 
 SPEED_MEDIUM_HIGH = 'medium_high'
 SPEED_MAPPING = {
-    SPEED_OFF: 0,
-    SPEED_LOW: 25,
-    SPEED_MEDIUM: 50,
-    SPEED_MEDIUM_HIGH: 75,
-    SPEED_HIGH: 100
+    SPEED_OFF: 0.00,
```

Comment:
Updated to speed values used by Lutron over their Telnet interface for the fan controller.